src/microbe_stage/ProcessList.cs

```diff
@@ -78,6 +81,8 @@ private ChemicalEquation CreateEquation(StrictProcessDisplayInfoEquality process
         equation.ShowSpinner = ShowSpinners;
         equation.MarkRedOnLimitingCompounds = MarkRedOnLimitingCompounds;
 
+        equation.ToggleProcessPressed += HandleToggleProcess;
```

Comment:
This syntax should not be used with Godot signals as this leaks resources.

The old style `Connect` syntax must be used.